proxy/src/pqproto.rs

```diff
@@ -222,7 +222,7 @@ where
     }
 
     // read in our entire message.
-    buf.resize(len, 0);
+    buf.resize(len as usize, 0);
```

Comment:
nit: consider `usize::from(len)`

Reply:
Not supported since rust supports 16 bit platforms, so usize might only be 16bit.

Reply:
hm, honestly, they could offer target-specific From impls.